src/vg/civcraft/mc/civchat2/listeners/BetterShardsListener.java

```diff
@@ -26,12 +26,15 @@ public BetterShardsListener(CivChat2 plugin) {
 	public void PlayerChangeServer(PlayerChangeServerEvent event){
 		String playerName = NameAPI.getCurrentName(event.getPlayerUUID());
 		UUID playerToReplyUUID = chatMan.getPlayerReply(playerName);
-		if (playerToReplyUUID == null) {
-			return;
+		if (playerToReplyUUID != null) {
+			String playerToReply = NameAPI.getCurrentName(playerToReplyUUID);
+			if(playerToReply != null){
+				MercuryAPI.sendMessage(event.getServerTravelingTo(), "channel" + sep + "reply" + sep + playerName + sep + playerToReply, "civchat2");
```

Comment:
No. I actually forgot a player can be in a channel with a player and not only with a group. Currently this change only handles /r and group channels. I will add private channels as well.